ui/app/adapters/role-jwt.js

```diff
@@ -9,7 +10,7 @@ export default ApplicationAdapter.extend({
     let [path, role] = JSON.parse(id);
 
     let namespace = get(snapshot, 'adapterOptions.namespace');
-    let url = `/v1/auth/${path}/oidc/auth_url`;
+    let url = `/v1/auth/${encodePath(path)}/oidc/auth_url`;
     let redirect_uri = `${window.location.origin}${this.router.urlFor('vault.cluster.oidc-callback', {
       auth_path: path,
```

Comment:
Little sanity check, line 13 you encode line 15 you don't. 99.9% sure thats ok, but just to check.

Reply:
Oh yep, urlFor is going to want encoded since it's router-related - thanks!